trolldb/api/api.py

```diff
@@ -81,7 +79,7 @@ def run_server(config: Union[AppConfig, FilePath], **kwargs) -> None:
     """
     logger.info("Attempt to run the API server ...")
     if not isinstance(config, AppConfig):
-        config = parse_config_yaml_file(config)
+        config = parse_config(config)
```

Comment:
The fact that this is not covered by tests worries me. Is codecov wrong?
Otherwise, either we don't test the same way that we would run production code and needs to be fixed, or this is actually never used and can be removed.

Reply:
Done! This check has been now removed.